### ISSUE TITLE
Package request: ros-noetic-mavros ros-noetic-mavros-extras geographiclib

### ISSUE THREAD

**Issue by cakarmustactalone** [arch4edu/arch4edu#189]
First of all thanks for this repository to save me for 10 hours of building ros-noetic package. Gladly find this repo. However I need to open this issue for mavros-packages , in aur version of ros-noetic-mavros, whenever I try to run my pixhawk , I got error : 
`/opt/ros/noetic/lib/mavros/mavros_node: error while loading shared libraries: libPocoFoundation.so.91: cannot open shared object file: No such file or directory`

But output of this:
```
pacman --query --list poco | grep libPocoFoundation
poco /usr/lib/libPocoFoundation.so
poco /usr/lib/libPocoFoundation.so.94
```

There is probably a version mismatching in these libraries.  

Could you add them to your repo? 
Thanks for your effort btw!

**Comment by cybaol** [comment]
Plus, package `ros-noetic-class-loader` is out-of-date because of  the dependence of `poco`.

**Comment by cybaol** [comment]
Is there a mechanism to automatically rebuild obsolete packages when dependencies are updated? @petronny

**Comment by petronny** [comment]
> Is there a mechanism to automatically rebuild obsolete packages when dependencies are updated?

Yes, but it's still under developement yet. Currently we have https://mirrors.tuna.tsinghua.edu.cn/arch4edu/x86_64/arch4edu.links.tar.gz which stores all links to dynamic libraries.

**Comment by petronny** [comment]
7d0c03f849a267787152f589627e53b1dcce29be
